relay/network_graph/dijkstra_weighted.py

```diff
@@ -87,12 +20,14 @@ def find_path(G, source, target, get_fee, value, max_hops=None, max_fees=None):
         (n, d, _, v) = pop(fringe)
         if v in dist:
             continue  # already searched this node.
+        if v == ignore:
```

Comment:
What about calling it ignored_nodes and make it a list, so that also more than one node can be ignored.

Reply:
Because it just adds complexity while not adding any value, in my opinion.
If I want to ignore a list of node, it is going to be faster to modify the graph directly when I know which node to ignore, otherwise we add nodes to be ignored to the list to visit and actually visit them but do nothing when you visit them. (It would be inefficient)

Reply:
Okay